test.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+if [ ! -f "/usr/local/bin/codeclimate" ]
+then
+	curl -L https://github.com/codeclimate/codeclimate/archive/master.tar.gz | tar xvz -C /home/$USER/
+	(cd /home/$USER/codeclimate-* && sudo make install)
```

Comment:
what about using just `$HOME/` here instead of `/home/$USER/`?  i was just testing this out locally and i don't have a `/home/$USER` but i do have a `$HOME` that points to `/users/dself` and I think `$HOME` will likely produce the same results on a linux based EC2 if thats where this is supposed to run otherwise.

Reply:
Ah, this makes a lot more sense now why it didn't work on other machines. Strangely for me
`home/$USER == $HOME` 🤷 Probably a left over from some other scripts I was doing